styx-common/src/test/java/com/spotify/styx/state/RunStateTest.java

```diff
@@ -435,6 +435,46 @@ public void testAccumulatesMessages() throws Exception {
   }
 
   @Test
+  public void testAccumulatesMessagesWithInfo() throws Exception {
+    transitioner.initialize(RunState.fresh(WORKFLOW_INSTANCE));
+    transitioner.receive(eventFactory.triggerExecution("trig"));
+    transitioner.receive(eventFactory.info("Depleted resources"));
+    transitioner.receive(eventFactory.dequeue());
+    transitioner.receive(eventFactory.submit(EXECUTION_DESCRIPTION));
+    transitioner.receive(eventFactory.submitted(TEST_EXECUTION_ID_1));
+    transitioner.receive(eventFactory.started());
+    transitioner.receive(eventFactory.terminate(20));
+    transitioner.receive(eventFactory.retryAfter(0));
+    transitioner.receive(eventFactory.dequeue());
+    transitioner.receive(eventFactory.runError("Error"));
+
+    assertThat(
+        transitioner.get(WORKFLOW_INSTANCE).data().messages(),
+        contains(
+            StateData.Message.create(StateData.MessageLevel.WARNING, "Depleted resources"),
+            StateData.Message.create(StateData.MessageLevel.WARNING, "Exit code: 20"),
+            StateData.Message.create(StateData.MessageLevel.ERROR, "Error")
+        ));
+  }
+
+  @Test
+  public void testUnknownResource() throws Exception {
```

Comment:
This test seems redundant. It only tests that run error adds a message, 2which there already is a test for.